app/views/hyrax/file_sets/_permission_form.html.erb

```diff
@@ -26,11 +26,11 @@
 
 <div class="form-group row">
   <div id="new-user">
-    <p class="col-sm-12">Enter <%= t('hyrax.account_label') %> (one at a time)</p>
+    <p class="col-sm-12"><%= t('.enter', account_label: t('hyrax.account_label')) %></p>
     <p class="sr-only">Use the add button to give access to one <%= t('hyrax.account_label') %> at a time (it will be added to the list below).  Select the user, by name or <%= t('hyrax.account_label') %>. Then select the access level you wish to grant and click on Add this <%= t('hyrax.account_label') %> to complete adding the permission.</p>
     <div class="col-sm-5">
       <label for="new_user_name_skel" class="sr-only"><%= t('hyrax.account_label') %> (without the <%= t('hyrax.directory.suffix') %> part)</label>
-      <%= text_field_tag 'new_user_name_skel', nil %>
+      <%= text_field_tag 'new_user_name_skel', t('.user_search') %>
```

Comment:
why are you adding text into the skeleton template?

Reply:
What's a "skeleton template"? 

This is used for the permission form, and I wanted it to not use the default text. This allows for internationallization to work for the text.

Reply:
@jcoyne can you clarify your question? I don't think anyone understands the significance of "skeleton template", and searching for related documentation hasn't been fruitful?

Reply:
the skeleton template is what the javascript uses to make a new row.  Since it's a new row, it should always be `nil`, right?  Why stick a translation on this particular line?

Reply:
This is the value of the form field, so it's not appropriate to put "Search for a user" here. That should either be in a label or in a placeholder attribute.